content/en/tests/setup/java.md

```diff
@@ -424,6 +433,33 @@ static Stream<Arguments> randomArguments() {
 }
 ```
 
+### Test session name `DD_TEST_SESSION_NAME`
+
+Use `DD_TEST_SESSION_NAME` to define the test session name for your tests (`test_session.name` tag). Use this to identify a group of tests. Examples of values for this tag would be:
+
+- `unit-tests`
+- `integration-tests`
+- `smoke-tests`
+- `flaky-tests`
+- `ui-tests`
+- `backend-tests`
+
+If `DD_TEST_SESSION_NAME` is not specified, the default value used is a combination of:
+
+- CI job name
+- Command used to run the tests (such as `yarn test`)
```

Comment:
I think something Java-specific, like `mvn test` or `./gradlew test`, would look more natural here :)

Reply:
oh sry I wanted to do this but I missed it, thanks